### PR TITLE
Upgrade Inventory UI dependencies, looks and usability

### PR DESCRIPTION
**Pull request by Morlack**

### CODE REVIEW

**Comment by Morlack** [comment]
Added room overview and replaced the manual UUID action with a button. Also, added Axon favicon, page title and Axon colors

image

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/web/rest/RoomQueryController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class RoomQueryController {
+
+    private final ReactorQueryGateway reactorQueryGateway;
+
+    public RoomQueryController(ReactorQueryGateway reactorQueryGateway) {
+        this.reactorQueryGateway = reactorQueryGateway;
+    }
+
+    @GetMapping(path = "/rooms")
+    public Mono<List<RoomOverviewData>> roomAvailability() {
```
Comment (idugalic) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/web/rest/RoomQueryController.java:40]: should we rename this method to `allRooms` (Availability is a valid concept only in Booking context, not in Inventory)?

**Review by idugalic — APPROVED** [review]
Looks great! Thanks!

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *  limitations under the License.
+ */
+
+package io.axoniq.demo.hotel.inventory.query;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface RoomEntityRepository extends JpaRepository<RoomEntity, String> {
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java:22]: Can't we make the repo package-private?

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java:2]: The copyright is just "a little" off ;-)

Comment (smcvb) [reply inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java:2]: Curious: what's the custom with copyright notices for js/ts/tsx files and such?

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.stream.Collectors;
+
+@Component
+@ProcessingGroup("room")
+class RoomHandler {
+
+    private final RoomEntityRepository roomEntityRepository;
+
+
+    private RoomOverviewData convert(RoomEntity entity) {
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java:38]: Nit: I'd likely make this method `static` myself.

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@ProcessingGroup("room")
+class RoomHandler {
+
+    private final RoomEntityRepository roomEntityRepository;
+
+
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java:37]: Super nit: I think we can drop one of these empty lines.

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    void on(RoomAddedToBookingSystemEvent event) {
+        RoomEntity entity = this.roomEntityRepository.getById(event.getRoomId().toString());
+        entity.setAddedToBooking(true);
+        this.roomEntityRepository.save(entity);
+    }
+
+    @QueryHandler
+    List<RoomOverviewData> on(FindRooms query) {
+        return this.roomEntityRepository.findAll().stream().map(this::convert).collect(Collectors.toList());
+    }
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java:69]: Matter of taste I think, but I'd draft this method up like so:
```suggestion
    List<RoomOverviewData> handle(FindRooms query) {
        return this.roomEntityRepository.findAll()
                                        .stream()
                                        .map(this::convert)
                                        .collect(Collectors.toList());
    }
```

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java [hunk 7]
```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java:2]: The copyright is just "a little" off ;-)

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/web/rest/RoomQueryController.java [hunk 8]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```
Comment (smcvb) [inline inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/web/rest/RoomQueryController.java:2]: The copyright is just "a little" off ;-)

File: inventory/src/main/kotlin/io/axoniq/demo/hotel/inventory/query/api/queries.kt [hunk 9]
```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```
Comment (smcvb) [inline inventory/src/main/kotlin/io/axoniq/demo/hotel/inventory/query/api/queries.kt:2]: The copyright is just "a little" off ;-)

File: inventory/src/main/kotlin/io/axoniq/demo/hotel/inventory/query/entities.kt [hunk 10]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```
Comment (smcvb) [inline inventory/src/main/kotlin/io/axoniq/demo/hotel/inventory/query/entities.kt:2]: The copyright is just "a little" off ;-)

**Review by smcvb — COMMENTED** [review]
Bunch of small-fry comments

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java (hunk 3 above)
Comment (Morlack) [reply inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java:2]: I don't know. What is Axon's policy? I have never understood the use of license headers (and thus updating them every year), while there is a LICENSE file in place, distributed with the code.

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java (hunk 4 above)
Comment (Morlack) [reply inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java]: Done

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java (hunk 3 above)
Comment (Morlack) [reply inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java]: Updated all the copyrights

File: inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java (hunk 6 above)
Comment (Morlack) [reply inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomHandler.java]: Reformatted

**Review by YvonneCeelie — APPROVED** [review]
LGTM

**Pull request merged**